src/main/java/com/redhat/cloud/policies/app/EnvironmentFlags.java

```diff
@@ -0,0 +1,36 @@
+package com.redhat.cloud.policies.app;
+
+import io.quarkus.runtime.Startup;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.jboss.logging.Logger;
+
+import javax.annotation.PostConstruct;
+import java.lang.reflect.Field;
+
+@Startup
+public class EnvironmentFlags {
+
+    private static final Logger LOG = Logger.getLogger(EnvironmentFlags.class);
+
+    @PostConstruct
+    public void init() {
+        try {
+            LOG.info("Environment flags in use:");
+            for (Field field: EnvironmentFlags.class.getDeclaredFields()) {
+                ConfigProperty configAnnotation = field.getAnnotation(ConfigProperty.class);
+                if (configAnnotation != null) {
+                    LOG.infof("  - %s: %s", configAnnotation.name(), field.get(this));
+                }
+            }
```

Comment:
Do you plan on adding more fields later?

Reply:
No plans on specific fields yet, but probably we could have more - I can remove this one for now if that's better.